src/service/resources-impl.js

```diff
@@ -99,9 +99,10 @@ export class Resources {
     /**
      * We want to do some work in the first pass after
      * the document is ready.
-     * @private {boolean}
+     * We also adjust the timeout penalty shortly after the first pass.
+     * @private {number}
      */
-    this.firstPassAfterDocumentReady_ = true;
+    this.firstVisibleTime_ = -1;
```

Comment:
Not a huge fun of combining concepts in one variable. From the sound of it "first visible" and "first after document ready" are not the same thing.